src/app/routes/topic/getInitialData/index.test.js

```diff
@@ -37,9 +37,17 @@ const optHeaders = { 'ctx-service-env': 'live' };
 describe('get initial data for topic', () => {
   const agent = { ca: 'ca', key: 'key' };
   const getAgent = jest.fn(() => agent);
+  const originalApplicationEnvironment = process.env.SIMORGH_APP_ENV;
+
+  beforeEach(() => {
+    process.env.SIMORGH_APP_ENV = 'live';
```

Comment:
could you explain why we're doing this? Worried that if it's important someone will remove at some point!

I get why we'd want to reset _after_ each. Just not why we set to live _before_

Reply:
If I don't do this, then `getInitialData` thinks the environment is local, and therefore will never attempt to fetch from the BFF. I think I could set the value to `test` (instead of `live`) and it would have the same effect, but basically, if I don't do this, then many of the tests will fail. I can change it to `test` instead of `live` if that helps?